src/config/mod.rs

```diff
@@ -1192,15 +1270,20 @@ impl RaftCfConfig {
     }
 }
 
-#[derive(Clone, Serialize, Deserialize, PartialEq, Debug)]
+#[derive(Clone, Serialize, Deserialize, PartialEq, Debug, OnlineConfig)]
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct TitanDbConfig {
+    #[online_config(skip)]
     pub enabled: Option<bool>,
+    #[online_config(skip)]
     pub dirname: String,
+    #[online_config(skip)]
     pub disable_gc: bool,
+    #[online_config(skip)]
     pub max_background_gc: i32,
     // The value of this field will be truncated to seconds.
+    #[online_config(skip)]
```

Comment:
```suggestion
#[derive(Clone, Serialize, Deserialize, PartialEq, Debug)]
#[serde(default)]
#[serde(rename_all = "kebab-case")]
pub struct TitanDbConfig {
    pub enabled: Option<bool>,
    pub dirname: String,
    pub disable_gc: bool,
    pub max_background_gc: i32,
    // The value of this field will be truncated to seconds.
```

FYI, `TitanDbConfig` has already been marked with `#[online_config(skip)]`, ref: https://github.com/tikv/tikv/blob/b53e5f79efe8873719ecae2c7904651d30e09e3c/src/config/mod.rs#L1866-L1867